pkg/costmodel/costmodel.go

```diff
@@ -487,21 +469,29 @@ func (cm *CostModel) ComputeCostData(cli prometheusClient.Client, cp costAnalyze
 				// recreate the key and look up data for this container
 				newKey := NewContainerMetricFromValues(ns, podName, containerName, pod.Spec.NodeName, clusterID).Key()
 
-				RAMReqV, ok := RAMReqMap[newKey]
-				if !ok {
-					klog.V(4).Info("no RAM requests for " + newKey)
-					RAMReqV = []*util.Vector{{}}
+				// k8s.io/apimachinery/pkg/api/resource/amount.go for explanations of
+				// the types that memory/cpu requests are
+				ramRequestBytes := container.Resources.Requests.Memory().Value()
+				RAMReqV := []*util.Vector{
```

Comment:
I believe it is okay to only set `Value` because the main consumer of this data [only uses Value](https://github.com/kubecost/cost-model/blob/mmd-ccd-reduce-prom/pkg/costmodel/metrics.go#L936). The [only other consumer](https://github.com/kubecost/cost-model/blob/mmd-ccd-reduce-prom/pkg/costmodel/router.go#L421) is just an API handler.

Reply:
This seems fine, but let's make sure we shake this out in testing by deploying this and making a note in the review